app/views/uniform/fragments/returnVariationDifferences.scala.html

```diff
@@ -24,143 +24,134 @@
 @(key: String, variation: ReturnVariationData, showChangeLinks: Boolean = true, broughtForward: Option[BigDecimal] = None)(implicit m: Messages)
 
 @cost(key: String, value: BigDecimal) = {
-    @defining(
-        key match {
-            case a if a == "own-brands-packaged-at-own-sites" || a == "packaged-as-a-contract-packer" || a == "brought-into-uk" => 1
-            case b if b == "claim-credits-for-exports" || b == "claim-credits-for-lost-damaged" => -1
-            case _ => 0
-        }
-    ) { multiplier =>
-      @format_money(value * multiplier)
-    }
+@defining(
+key match {
+case a if a == "own-brands-packaged-at-own-sites" || a == "packaged-as-a-contract-packer" || a == "brought-into-uk" => 1
+case b if b == "claim-credits-for-exports" || b == "claim-credits-for-lost-damaged" => -1
+case _ => 0
+}
+) { multiplier =>
+@format_money(value * multiplier)
+}
 }
 
 @table_head(title: String, changeLink: String)(rowContent: Html) = {
-    <table class="check-your-answers">
-        <thead>
-            <tr>
-                <th @if(showChangeLinks) {colspan="3"} else {colspan="4"} >
-                    <p class="heading-medium">
-                    @Messages(s"$key.$title")
-                    </p>
-                </th>
-                @if(showChangeLinks) {
-                    <th class="change-answer change-link-title" colspan="1">
-                        <a href="@changeLink">
-                            <p class="edit-change-link">@Messages("sdil.declaration.change-link")</p>
-                            <span class="visuallyhidden">@Messages(s"$key.change.$title")</span>
-                        </a>
-                    </th>
-                }
-            </tr>
-        </thead>
-        <tbody>
-        @rowContent
-        </tbody>
-    </table>
+<table class="check-your-answers">
+    <caption  @if(showChangeLinks) {colspan="4"} else {colspan="3"} class="heading-medium">@Messages(s"$key.$title")
+        @if(showChangeLinks) {
+        <a href="@changeLink">
+            <p class="edit-change-link change-answer change-link-title change-link-padding">@Messages("sdil.declaration.change-link")</p>
```

Comment:
The change link need to be always aligned to the right, now it follows straight after the title, so it is in varying places accrosss the page. If you view the page with all all of the possible return values changed, you'll see this.